processing_r/processing/algorithm.py

```diff
@@ -645,7 +645,13 @@ def build_import_commands(self,  # pylint: disable=too-many-locals,too-many-bran
                     layer_idx += 1
                 s += ')'
                 commands.append(s)
-            # TODO folder, file/html output paths should be set here
+
+        # TODO folder, file/html output paths should be set here
```

Comment:
This line can be revised now, right?

Reply:
Add folder destination path in input commands.